pkgs/development/libraries/unicorn-emu/default.nix

```diff
@@ -1,18 +1,25 @@
-{ stdenv, fetchurl, pkgconfig, python }:
+{ stdenv, fetchurl, pkgconfig, python, cmocka, hexdump, setuptools }:
```

Comment:
Don't add setuptools because it should be guaranteed it is part of the same interpreter.